apmapf.py

```diff
@@ -151,12 +212,16 @@ def timed_get_model(*args, **kwargs):
         logging.log(logging.INFO, 'Accepted by solver.')
 
     def run(self):
-        R = len(self.content['starts'])        # robots
-        N = self.content['N']        # grid size
+        R = len(self.content['starts']) # robots
+        N = self.content['N']           # grid size
 
+        # We use a breadth-first search to determine the horizon, i.e., 
```

Comment:
Is this explanation correct?

Reply:
Yes, we use BFS from each starting location to determine a lower bound for the makespan of the attack-proof MAPF plan.